assets/js/components/Dashboard/PaMessagesPage.tsx

```diff
@@ -78,7 +133,7 @@ const PaMessageTable: ComponentType<PaMessageTableProps> = ({
       </table>
       {paMessages.length == 0 && (
         <div className="pa-message-table__empty">
-          There are no active PA/ESS Messages.
+          There are no PA/ESS Messages matching the current filters.
```

Comment:
Should we add a simple text message for the error/loading states of SWR as well? In the video it does look a bit odd that the table briefly disappears when loading new messages.

Reply:
I've asked Mary how she'd like these loading states to work [[slack](https://mbta.slack.com/archives/C039ARUM48H/p1723237320826709)]

Reply:
Okay Mary asked for a spinner in the table when loading new data. You could still see a pop of that loading state whenever you clicked on a new filter though. So I've also added a 250ms delay to showing the loading state at all. This is something I've done in the past and in my opinion makes the application feel snappier.

<details>
<summary>Fast network</summary>

https://github.com/user-attachments/assets/b6aa6baf-7238-4de7-90bb-4abfb4fc2e95

</details>

<details>
<summary>Slow network</summary>

https://github.com/user-attachments/assets/d74f23b2-da42-4026-89fd-00c12322dac1

</details>

Reply:
I do kind of prefer my callback-less approach but it would totally work either way. Not a hard thing to change later on if we feel like we need to for some reason.